scripts/cmake-presets/goldfinger.json

```diff
@@ -27,7 +29,7 @@
         "CELERITAS_BUILD_DEMOS": {"type": "BOOL",   "value": "OFF"},
         "CELERITAS_BUILD_TESTS": {"type": "BOOL",   "value": "OFF"},
         "CELERITAS_USE_JSON":    {"type": "BOOL", "value": "ON"},
-        "CMAKE_CXX_INCLUDE_WHAT_YOU_USE": "$env{IWYU_ROOT}/bin/include-what-you-use;-Xiwyu;--no_fwd_decls;-Xiwyu;--no_comments;-Xiwyu;--transitive_includes_only"
+        "CMAKE_CXX_INCLUDE_WHAT_YOU_USE": "/opt/homebrew/bin/include-what-you-use;-Xiwyu;--no_fwd_decls;-Xiwyu;--no_comments;-Xiwyu;--transitive_includes_only"
```

Comment:
You might consider updating the git log to note why this is an improvement (eg. the `$env` was not working or is no longer necessary because the value is now "fixed")

Reply:
This is just for my personal mac... I switched from a spack installation to a homebrew one.